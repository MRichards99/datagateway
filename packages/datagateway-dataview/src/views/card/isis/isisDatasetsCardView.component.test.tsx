import { createShallow } from '@material-ui/core/test-utils';

describe('ISIS Datasets - Card View', () => {
  let shallow;

  beforeEach(() => {
    shallow = createShallow();
  });

  it.todo('renders correctly');
});
