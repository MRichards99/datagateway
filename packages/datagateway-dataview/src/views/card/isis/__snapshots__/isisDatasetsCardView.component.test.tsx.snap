// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ISIS Datasets - Card View renders correctly 1`] = `
<ContextProvider
  value={null}
>
  <ISISDatasetsCardView
    addToCart={[Function]}
    cartItems={Array []}
    data={
      Array [
        Object {
          "CREATE_TIME": "2019-07-23",
          "ID": 1,
          "INVESTIGATION_ID": 1,
          "MOD_TIME": "2019-07-23",
          "NAME": "Test 1",
          "SIZE": 1,
        },
      ]
    }
    downloadData={[Function]}
    fetchCount={[Function]}
    fetchData={[Function]}
    fetchDetails={[Function]}
    investigationId="1"
    loadedCount={true}
    loadedData={true}
    pushFilters={[Function]}
    pushPage={[Function]}
    pushQuery={[Function]}
    query={
      Object {
        "filters": Object {},
        "page": null,
        "results": null,
        "search": null,
        "sort": Object {},
        "view": null,
      }
    }
    removeFromCart={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    totalDataCount={1}
  />
</ContextProvider>
`;
